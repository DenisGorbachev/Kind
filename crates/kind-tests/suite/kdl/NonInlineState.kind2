#kdl_state = MyFn.state
#keep
MyFn : Data.U60 {
  1
}

MyFn.state : Data.U60