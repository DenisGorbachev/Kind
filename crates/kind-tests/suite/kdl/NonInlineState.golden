   ERROR  This entry should only have one rule.

      /--[suite/kdl/NonInlineState.kind2:7:1]
      |
    6 |    
    7 |    MyFn.state : Data.U60
      |    v--------------------
      |    \Here!


